application/src/main/java/org/togetherjava/tjbot/commands/tophelper/TopHelpersCommand.java

```diff
@@ -143,29 +146,30 @@ private static void handleTopHelpers(Collection<TopHelperResult> topHelpers,
                     userIdToMember.get(topHelper.authorId())))
             .toList();
 
-        String message =
-                "```java%n%s%n```".formatted(dataTableToString(topHelpersDataTable, timeRange));
+        String message = """
+                ```java
+                // for %s
```

Comment:
What does this mean, I need some pictures to understand the context

Reply:
https://i.imgur.com/bl8MbIP.png

Reply:
And some users with very long names, lets say an user named `FooBarBazHelloWorld` would now be abbreviated to `FooBarBazHel...` (we have the ID on the left anyways)